examples/quick-start-guide.md

```diff
@@ -0,0 +1,109 @@
+# Quick Start Guide
+
+This section helps you to quickly set up and run Ballerina so that you can use it for various solutions. Here we 
+
+## Install Ballerina
+
+1. [Download](https://ballerina.io/downloads) Ballerina for your Operating System. 
+1. Follow the instructions given in the [Ballerina Getting Started page](#https://ballerina.io/learn/getting-started/) to set it up. 
+
+> **Note**: Throughout this documentation, `<ballerina_home>` refers to the directory in which you just installed Ballerina.
+
+## Start a Project, Run a Service, and Invoke It
+
+Start your project by navigating to a directory of your choice and running the following command.
+
+```bash
+$ ballerina init
+```
+
+You see a response confirming that your project is initialized. This automatically creates a typical Hello World service for you in your directory. A Ballerina service represents a collection of network accessible entry points in Ballerina. A resource within a service represents one such entry point. The generated sample service exposes a network entry point on port 9090.
+
+You can run the service using the `ballerina run` command.
+
+```bash
+$ ballerina run hello_service.bal
```

Comment:
Before running the service we need to run ballerina create to create a project using a template. We can use the service template in this example.
```bash
ballerina create MyProject -t service
```

after that, we can execute the command 
```bash
ballerina run MyProject
```